.github/workflows/lint-squid.yaml

```
@@ -23,7 +23,7 @@ jobs:
         key: ${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}
 
     - name: install packages
-      run: npm ci
+      run: yarn install --immutable
```

This is for subsquid that only officially supports npm, so we need to keep it that way I'd say.
```suggestion
      run: npm ci
```